pandas/core/series.py

```diff
@@ -3635,6 +3636,59 @@ def reorder_levels(self, order):
         result.index = result.index.reorder_levels(order)
         return result
 
+    def explode(self) -> "Series":
+        """
+        Create new Series expanding a list-like column.
+
+        .. versionadded:: 0.25.0
+
+        Returns
+        -------
+        exploded: Series
```

Comment:
I think a space is needed before and after colon for docstring standard

Reply:
do we have a check on this?

Reply:
Hmm looks like only for Parameters:

https://github.com/pandas-dev/pandas/blob/acccdcc6c85d2331f1dd3ae14977ea861437f2d8/scripts/validate_docstrings.py#L117

Though typically with Returns only have the type and not name of the parameter being returned, so maybe less important there. @datapythonista

Reply:
The standard for `Returns` is to not have a name unless a tuple is returned and we have different values. I think we validate those with the script, but probably not activated in the CI yet because the cases that are wrong.